extensions/nns/src/main.rs

```diff
@@ -15,6 +17,11 @@ pub struct NnsOpts {
     /// `dfx nns` subcommand arguments.
     #[clap(subcommand)]
     subcmd: SubCommand,
+
+    // global args have to be wrapped with Option for now: https://github.com/clap-rs/clap/issues/1546
+    /// Path to cache of DFX which executed this extension.
+    #[arg(long, global = true)]
```

Comment:
Have you considered allowing to pass this as well through an environment variable?
```suggestion
    #[arg(long, env = "DFX_CACHE_PATH", global = true)]
```